Client/Frontend/Browser/TabLocationView.swift

```diff
@@ -153,18 +177,37 @@ class TabLocationView: UIView {
         addSubview(urlTextField)
         addSubview(lockImageView)
         addSubview(readerModeButton)
+        addSubview(pageOptionsButton)
+        addSubview(separatorLine)
 
         lockImageView.snp.makeConstraints { make in
             make.size.equalTo(24)
             make.centerY.equalTo(self)
             make.leading.equalTo(self).offset(9)
         }
 
+        pageOptionsButton.snp.makeConstraints { make in
+            make.centerY.equalTo(self)
+            make.trailing.equalTo(self)
+            make.width.equalTo(44)
+            make.height.equalTo(self)
+        }
+        
+        separatorLine.snp.makeConstraints { make in
+            make.width.equalTo(1)
+            make.height.equalTo(26)
+            make.trailing.equalTo(pageOptionsButton.snp.leading)
+            make.centerY.equalTo(self)
+        }
+        
         readerModeButton.snp.makeConstraints { make in
             make.centerY.equalTo(self)
-            make.trailing.equalTo(self).offset(-9)
+            make.trailing.equalTo(separatorLine.snp.leading).offset(-9)
             make.size.equalTo(24)
         }
+        
+
+    
```

Comment:
nit: newlines

Reply:
hmm. Odd is swiftlint not running. It would have pointed this out.